src/clj/reviews_next/db/db_ops.clj

```diff
@@ -0,0 +1,34 @@
+(ns reviews-next.db.db_ops
+  (:require
+          [clojure.pprint :as pp]
+          [clojure.java.jdbc :refer :all]))
+
+(def db
+  {:classname   "org.sqlite.JDBC"
+   :subprotocol "sqlite"
+   :subname     "reviews_next.db"})
```

Comment:
Ideally the db connection configuration should sit in config.clj
Also it needs to be namespaced by the environment. So for instance, currently it seems like the db_ops namespace will always use the reviews_next.db even while it is running tests.